e2e/cli/command/util.go

```diff
@@ -0,0 +1,55 @@
+package command
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"runtime"
+	"time"
+
+	getter "github.com/hashicorp/go-getter"
+	"github.com/hashicorp/nomad/helper/discover"
+)
+
+// Fetches the nomad binary and returns the temporary directory where it exists
+func fetchBinary(bin string) (string, error) {
```

Comment:
https://blog.golang.org/godoc-documenting-go-code

`fetchBinary fetches the nomad...`